Documentation/debugging/windows-instructions.md

```diff
@@ -0,0 +1,93 @@
+Debugging CoreFX on Windows
+==========================
+
+You can Debug .NET Core via Visual Studio or WinDBG.
+
+For Visual Studio debugging, follow the instructions at [Debugging tests in Visual Studio](https://github.com/dotnet/corefx/blob/master/Documentation/building/windows-instructions.md) to run and debug tests.
```

Comment:
On the referenced windows-instructions page, can you add a link back to this page as well?